upgrade_tests/test_scenarios/test_capsule.py

```diff
@@ -43,9 +43,9 @@ class Scenario_capsule_sync(TestCase):
     """
     cls_name = 'Scenario_capsule_sync'
     sat_host = get_satellite_host()
```

Comment:
Can we have this in init ?

Reply:
Not required as its declared at a class scope and will be initialized when the class is called, No need of __init__.